tests/bdd/features/helm_chart/install.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Install Mayastor
+  Install Mayastor on a kubernetes cluster, using helm
+
+  Background:
+    Given: A Kubernetes cluster
+
+  Scenario: Mayastor helm chart install
+    Given worker nodes on the kubernetes cluster are labelled with the label 'openebs.io/engine=mayastor'
+    And a kubernetes namespace to install the Mayastor helm chart
+    When the Mayastor helm chart is installed with callhome disabled and at most 3 replicas of etcd
+    Then the helm chart release is in 'deployed' state
+    And one api-rest Kubernetes Deployment is Available
+    And one agent-core Kubernetes Deployment is Available
+    And one csi-controller Kubernetes Deployment is Available
+    And one operator-diskpool Kubernetes Deployment is Available
+    And one localpv-provisioner Kubernetes Deployment is Available
+    And one csi-node Kubernetes DaemonSet for which all replicas are Available
+    And one ha-node Kubernetes DaemonSet for which all replicas are Available
+    And one io-engine Kubernetes DaemonSet for which all replicas are Available
+    And one promtail Kubernetes DaemonSet for which all replicas are Available
+    And one etcd Kubernetes StatefulSet for which all replicas are Available
```

Comment:
Do we need to go to such detail, perhaps we should simply verify that we can create a pod which references a volume here?

Reply:
Resolving this conversation, because this test is no longer a part of this PR.